pack.yaml

```diff
@@ -5,7 +5,7 @@ runner_type: "python-script"
 description: Terraform integrations
 keywords:
   - terraform
-version: 2.0.0
+version: 2.0.1
```

Comment:
Needs a Changelog record for 2.0.1 here:
https://github.com/StackStorm-Exchange/stackstorm-terraform/blob/master/CHANGES.md

Reply:
@armab  - changelog updated.